### DISCUSSION TITLE
⛔ Remove query strings for better performance

### DISCUSSION THREAD

**Discussion by jazir555** [elementor/elementor#25650] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Elementor appends query strings to static resources, which limit cachability. If an option to strip query strings was added, it would improve pagespeed.

### Describe the solution you'd like.

Add a toggle to remove query strings from css and js files

### Describe alternatives you've considered.

Query string removal plugins do not work

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @jazir555,

Thank you for submitting a feature request! 🙏 

### ⛔ 🚀 After reviewing your request, we have decided not to add this to our roadmap for the time being.

Query strings are essential to allow Elementor's CSS to be updated when there are changes to styles in your documents. Without query strings, users would not see any changes.

Please feel free to contact us again with other feature suggestions. 😉 

Best regards
